pkg/operator/ceph/cluster/version_test.go

```diff
@@ -109,6 +110,57 @@ func TestDiffImageSpecAndClusterRunningVersion(t *testing.T) {
 	m, err = diffImageSpecAndClusterRunningVersion(fakeImageVersion, dummyRunningVersions5)
 	assert.NoError(t, err)
 	assert.False(t, m)
+
+	// 6 test - spec version and running cluster have same build but different build suffix --> we upgrade
+	fakeImageVersion = cephver.CephVersion{Major: 14, Minor: 2, Extra: 11, Build: 139,
+		BuildSuffix: "el8cp", Image: "5c0dc966af809fd1d429ec7bac48962a746af243"}
+	fakeRunningVersions = []byte(`
+		{
+			"overall": {
+				"ceph version 14.2.11-139.0.hotfix.bz1959254.el8cp (5c0dc966af809fd1d429ec7bac48962a746af243) nautilus (stable)": 2
```

Comment:
So this `139.0.hotfix.bz1959254` is the `Build`? and `el8cp` the `BuildSuffix`?

Reply:
`139` is the build and anything after that is the build suffix `0.hotfix.bz1959254.el8cp` in this case.  The implementation in `master` branch only checks till `14.2.11-139`  .  [Reference](https://github.com/rook/rook/blob/52a9dfc26d65dd7b355eb23f2707c3aa65d20a62/pkg/operator/ceph/version/version.go#L66)